.github/workflows/make_logos.yaml

```diff
@@ -30,7 +30,7 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
```

Comment:
Again, unrelated, but the [Actions](https://github.com/WISVCH/icpc-logos/actions/runs/11439948342) page shows warning that this version is deprecated. I believe the latest is `@v4`?

(same for other jobs, of course)

Reply:
hmm, I updated it, but the actions page still claims we use it. Did I miss any?